docs/querying/api.md

```diff
@@ -206,6 +206,35 @@ $ curl 'http://localhost:9090/api/v1/query_range?query=up&start=2015-07-01T20:10
 }
 ```
 
+## Formatting query expressions
```

Comment:
Will it be nice to add that "prettified expressions will follow the specifications as mentioned [here](https://github.com/prometheus/prometheus/blob/main/promql/parser/prettier_rules.md)"?

Reply:
I would skip it for now because not all things in that spec is user digestible. For example `max_characters_per_line`. It is more of a developer reference. I am not sure if we need to have a user facing spec for this (for now. If we do in future, we need to improve it).

Reply:
(In a user facing doc, max_characters_per_line will be replaced with some actual number for example)